lib/i18n-ai/railtie.rb

```diff
@@ -55,7 +55,14 @@ def generate_translations(locales_file)
 
           translated_text = response["choices"][0]["message"]["content"]
           match_data = translated_text.match(/```yaml(.*?)```/m)
-          str = match_data ? match_data[1].strip : nil
```

Comment:
currently not working because `str` is used in another line below:

https://github.com/narralabs/i18n-ai/blob/3f8159f707388e7660f66b478b3755539306c520/lib/i18n-ai/railtie.rb#L69

do you mind fixing? this should be easier to catch once we add more tests.

thank you.

Reply:
@westoque Got it, thanks!